tests/test_dynamodb.py

```diff
@@ -222,3 +223,107 @@ def test_dynamodb_enable_stream(self):
+            'resource': 'dax',
+            'filters': [{
+                'type': 'security-group',
+                'key': 'GroupName',
+                'value': 'default'}]
+        }, session_factory=session_factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['ClusterName'], 'c7n-dax')
+
+    def test_tagging(self):
+        session_factory = self.replay_flight_data(
+            'test_dax_add_tags')
+        p = self.load_policy({
+            'name': 'dax-resources',
+            'resource': 'dax',
+            'filters': [{'tag:Required': 'absent'}],
+            'actions': [{
+                'type': 'tag',
+                'tags': {'Required': 'Required'}
+            }]}, session_factory=session_factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['ClusterName'], 'c7n-dax')
+        client = session_factory(region='us-east-1').client('dax')
+        tags = client.list_tags(ResourceName=resources[0]['ClusterArn'])['Tags']
+        print(tags[0]['Value'])
+        self.assertEqual(tags[0]['Value'], 'Required')
+
+    def test_remove_tagging(self):
+        session_factory = self.replay_flight_data(
+            'test_dax_remove_tags')
+        p = self.load_policy({
+            'name': 'dax-resources',
+            'resource': 'dax',
+            'filters': [{'tag:Required': 'present'}],
+            'actions': [{
+                'type': 'remove-tag',
+                'tags': ['Required']
+            }]}, session_factory=session_factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['ClusterName'], 'c7n-dax')
+        client = session_factory(region='us-east-1').client('dax')
+        tags = client.list_tags(ResourceName=resources[0]['ClusterArn'])['Tags']
+        self.assertEqual(tags, [{'Key': 'Name', 'Value': 'c7n-dax-test'}])
+
+    def test_mark_for_op(self):
+        dtnow = datetime.datetime(year=2018, month=5, day=4, hour=16, minute=0)
+        session_factory = self.replay_flight_data(
+            'test_dax_mark_for_op')
+        p = self.load_policy({
+            'name': 'dax-resources',
+            'resource': 'dax',
+            'filters': [
+                {'tag:custodian_cleanup': 'absent'},
+                {'tag:Required': 'absent'}],
+            'actions': [{
+                'type': 'mark-for-op',
+                'tag': 'custodian_cleanup',
+                'msg': "Missing tag Required: {op}@{action_date}",
+                'op': 'delete',
+                'days': 7}]}, session_factory=session_factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['ClusterName'], 'c7n-dax')
+        client = session_factory(region='us-east-1').client('dax')
+        dtact = (dtnow + timedelta(days=7)).strftime('%Y/%m/%d %H%M UTC')
```

Comment:
hmm.. I thought we only did hours/minutes in the tag value if we specified hours in the action.